src/components/manifold-subscription-list/manifold-subscription-list.tsx

```diff
@@ -11,10 +11,13 @@ import query from './subscriptions.graphql';
 export class ManifoldSubscriptionList {
   @Element() el: HTMLElement;
 
+  @Event() ctaClick: EventEmitter;
+
   @Prop({ mutable: true }) connection?: Connection;
   @Prop({ mutable: true }) loading?: boolean = false;
   @Prop({ mutable: true }) errors?: GraphqlError[];
   @Prop({ mutable: true }) data?: SubscriptionsQuery;
+  @Prop() baseUrl?: string = '/subscriptions';
```

Comment:
You probably don't want to give this a default value. If this property is uninitialized, it should signal to the component that we don't want to navigate anywhere.